src/databricks/labs/ucx/source_code/redash.py

```diff
@@ -1,24 +1,130 @@
+import logging
+from collections.abc import Iterator
+from dataclasses import replace
+
+from databricks.labs.blueprint.installation import Installation
+
 from databricks.sdk import WorkspaceClient
-from databricks.sdk.service.sql import Query
+from databricks.sdk.service.sql import Query, Dashboard
+from databricks.sdk.errors.platform import DatabricksError
 
-from databricks.labs.ucx.source_code.base import Fixer
+from databricks.labs.ucx.hive_metastore.migration_status import MigrationIndex
+from databricks.labs.ucx.source_code.base import CurrentSessionState
+from databricks.labs.ucx.source_code.queries import FromTable
+
+logger = logging.getLogger(__name__)
 
 
 class Redash:
-    def __init__(self, fixer: Fixer, ws: WorkspaceClient):
-        self._fixer = fixer
+    MIGRATED_TAG = "migrated by UCX"
```

Comment:
```suggestion
    MIGRATED_TAG = "Migrated by UCX"
```

Similarly to the "Created by UCX" tag used for other resources